src/bokeh/models/graphs.py

```diff
@@ -82,11 +85,11 @@ class StaticLayoutProvider(LayoutProvider):
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
-    # TODO: Seq(Any).length == 2
-    graph_layout = Dict(Int, Seq(Any), default={}, help="""
-    The coordinates of the graph nodes in cartesian space. The dictionary
-    keys correspond to a node index and the values are a two element sequence
-    containing the x and y coordinates of the node.
+    graph_layout = Dict(Either(Int, String), OfLength(Seq(Float), 2), default={}, help="""
```

Comment:
The order of `OfLength(Seq(Float), 2)` makes for extremely awkward grammar. At a minimum, switching the order to `OfLength(2, Seq(Float))` would read more clearly, unless something like`Seq(Float).OfLength(2)` is attainable.

Reply:
> `OfLength(Seq(Float), 2)`

This is not far off from `len(seq) == 2`, which is what I was targeting. Maybe I should have used `Len(Seq(Float), 2)` instead? However, adding a method to container property type seems a very good idea.

Reply:
`Len` would be preferable to `OfLength` (to me at least)